include/dt-bindings/gpio/gpio.h

```diff
@@ -76,6 +76,36 @@
 
 /** @} */
 
+/**
+ * @name GPIO pin voltage flags
+ *
+ * Only applicable if SoC allows to configure pin voltage per individual pin.
+ *
+ * @{
+ */
+
+/**
+ * @cond INTERNAL_HIDDEN
+ *
+ * For internal use only, skip these in public documentation.
+ */
+#define GPIO_VOLTAGE_POS        6
+#define GPIO_VOLTAGE_MASK       (3U << GPIO_VOLTAGE_POS)
+/**
+ * @endcond
+ */
+
+/** Set pin at the default voltage level */
+#define GPIO_VOLTAGE_DEFAULT    (0U << GPIO_VOLTAGE_POS)
+/** Set pin voltage level at 1.8 V */
+#define GPIO_VOLTAGE_1P8        (1U << GPIO_VOLTAGE_POS)
+/** Set pin voltage level at 3.3 V */
+#define GPIO_VOLTAGE_3P3        (2U << GPIO_VOLTAGE_POS)
+/** Set pin voltage level at 5.0 V */
+#define GPIO_VOLTAGE_5P0        (3U << GPIO_VOLTAGE_POS)
```

Comment:
Why aren't those suffixes `1V8`, `3V3`, and `5V0`? Such notation seems to be more widely used in Zephyr.

Reply:
I have seen EEs use 1P8, 3P3 and 5P0 when the name is already clear this is a voltage. On the other hand, if the names were GPIO_LEVEL_* then it would not be clear and 1V8, 3V3 and 5V0 would be used to make it clear what the level is referring to. I can change this if it is desired to move away from this EE naming.

Reply:
I grepped the code and there are uses of both P and V.